src/main/java/com/tstepnik/planner/service/TaskService.java

```diff
@@ -53,6 +63,8 @@ public Task updateTask(Task task, Long taskId) {
         Task checkedTask = taskRepository.getOne(taskId);
         if (!checkedTask.getUserId().equals(user.getId())) {
             throw new AccessDeniedException("User with login " + user.getId() + " cannot edit task with id " + taskId);
+        } else if (checkedTask.getPlannedFor().isBefore(ZonedDateTime.now(ZoneId.of("UTC")))) {
+            throw new ExpiredTaskException("Time for finish this task had expired at: " + checkedTask.getPlannedFor() + " You can't do any changes on it now.");
```

Comment:
Ok, now I get how you want to use this exception but exception message seems a little bit too much. Exception messages are for developers, not users. So the last part shouldn't be here.